storybookjs/storybook
# Ember: Fix broken import

Following #21435.

Hopefully closes #20653 



## What I did

Ember [recommends to use `@glimmer/component` rather than `@ember/component`](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-classic-components.md). I switched the import and the package.json accordingly. I believe ember itself somehow import `@glimmer/component` so maybe the peer dependency is not needed. I couldn't launch the project myself locally so let's wait for CI check.

## How to test

- Create a fresh app Ember
```
npx --package ember-cli ember --version #ember-cli: 4.11.0
npx --package ember-cli ember new ember-quickstart
```
- Check Ember app works fine
```
cd ember-quickstart
npm start
```
- Install this storybook PR for Ember
```
npm install --save-dev yannbertrand/storybook#main
```
- Init Storybook
```
npx storybook --version #6.5.16
npx storybook init
```
- Hopefully this command now works and render something
```
npm run storybook
```

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
